File: ws/ullcu/U3/ws40ExeciseAge/src/main/java/utils/MongoDBConnection.java

package utils;

/**
 *
 * @author USUARIO
 */
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoDatabase;

public class MongoDBConnection {
private static final String URI = "mongodb+srv://daullcu:daullcu@cluster0.dxj9m.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0";
    private static final String DATABASE_NAME = "your_database_name";

    public static MongoDatabase getDatabase() {
        MongoClient client = MongoClients.create(URI);
        return client.getDatabase(DATABASE_NAME);
    }
}

File: EXAM/bonilla/u3/exam3_final/espe/edu/ec/utils/mongodb_connection.py
from pymongo import MongoClient
from pymongo.server_api import ServerApi

def get_database():
    uri = "mongodb+srv://rchapigo:1234@cluster0.0etpp.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0"
    client = MongoClient(uri, server_api=ServerApi('1'))
    db = client['BonillaStrategy']  
    return db['BonillaSortedArray']  
